pypeit/core/procimg.py

```diff
@@ -265,59 +266,104 @@ def gain_frame(amp_img, gain):
     return gain_img
 
 
-def rn_frame(datasec_img, gain, ronoise):
-    """ Generate a RN image
+def rn2_frame(datasec_img, gain, ronoise, units='e-', digitization=True):
```

Comment:
Thanks for the name change!

Reply:
Since digitisation noise computation is optional, maybe gain should be an optional argument as well. We don't need the gain if digitisation noise is not being computed.

Reply:
Yep, good call.  Changed.